integrationTests/multiShard/block/executingMiniblocks_test.go

```diff
@@ -373,18 +382,45 @@ func TestShouldProcessBlocksInMultiShardArchitecture(t *testing.T) {
 
 }
 
+func getBlockBody(
```

Comment:
function does not explain what it does.

Reply:
Agree, will be completely removed from this file when the refactoring is done. This func has code from TestNodeProcessor.SyncBlock()

Reply:
Still you should rename the function until the refactor is done. Just add a longer name, createBlockBodyFromOrderedMiniblocks or something similar.

Reply:
ok

Reply:
Done